src/scripts/eos-analysis_TEST

```diff
@@ -9,6 +9,16 @@ export EOS_BASE_DIRECTORY=$(mktemp -d)
 echo using PYTHON=${PYTHON} >&2
 echo using PYTHONPATH=${PYTHONPATH} >&2
 
+###########################################
+## Run the Analysis's steps automaticall ##
+###########################################
+echo running analysis steps test ... >&2
+$PYTHON ${SOURCE_DIR}/eos-analysis \
+    run \
+    -f ${SOURCE_DIR}/eos-analysis_TEST.d/analysis.yaml
+echo ... success >&2
```

Comment:
Should this test replace the other tests?

Reply:
I thought about this yesteday. I think it should not, since we are both testing the tasks themselves and the CLI.

Reply:
I was just afraid of time-consumption, but if we only test on a dry-run, it should be ok :)

Reply:
That's not such a good idea, since I want to test that this really works. It runs one optimization and four short markov chains, which is really not a big time consumption.